MAVProxy/modules/mavproxy_nokov/__init__.py

```diff
@@ -0,0 +1,127 @@
+"""
+use nokov data to provide motion capture data
+it works with nokov software
+"""
+
+import time
+
+from MAVProxy.modules.lib import mp_module
+from MAVProxy.modules.lib import mp_settings
+from MAVProxy.modules.mavproxy_nokov.nokov import nokovsdk
```

Comment:
where is nokovsdk?

Reply:
We plan to send the nokovsdk directly to the users who need it.Is that ok?

Reply:
The SDK has been committed.